### PR TITLE
allow custom stop tokens for OpenAI answer

### PR DESCRIPTION
**Pull request by barnes-b**
As far as I can tell, `answer`  doesn't do any special question/answer formatting, which confused me temporarily, suggest changing the description

### CODE REVIEW

**Comment by stuhlmueller** [comment]
Good points! 
- I think the `multiline` argument in the agent answer interface should be changed to `stop` more globally, i.e. also in [these places](https://github.com/oughtinc/ice/search?q=multiline).
- `answer` should probably be renamed to `generate` or `complete` (now that it doesn't do question-specific formatting). This also requires changing the primer.

I can do both of these on Tuesday, but if anyone else gets there before I do I would be grateful!

File: ice/agents/openai.py [hunk 1]
```diff
@@ -31,13 +31,14 @@ async def answer(
         prompt: str,
         multiline: bool = True,
         verbose: bool = False,
-        default: str = "",
```
Comment (alexmojaki) [inline ice/agents/openai.py:34]: I don't think this can be removed here unless it's removed in all agents.

**Comment by barnes-b** [comment]
I have read the CLA Document and I hereby sign the CLA

File: ice/agents/openai.py (hunk 1 above)
Comment (barnes-b) [reply ice/agents/openai.py:34]: ah ok, sounds plausible!

**Comment by stuhlmueller** [comment]
Implemented a more general version of this in #52

**Pull request closed without merging**